src/EditorFeatures/TestUtilities/CallHierarchy/CallHierarchyTestState.cs

```diff
@@ -256,8 +255,7 @@ internal void VerifyResult(CallHierarchyItem root, string searchCategory, string
         {
             this.SearchRoot(root, searchCategory, (CallHierarchyItem c) =>
                 {
-                    Assert.True(expectedCallers.Any());
-                    Assert.True(expectedCallers.Contains(ConvertToName(c)));
+                    Assert.Contains(ConvertToName(c), expectedCallers);
                 },
                 scope,
                 documents);
```

Comment:
Perhaps a test can be added where teh workspace is edited and then the search happens.